### ISSUE TITLE
kelpy = noob

### ISSUE THREAD

**Issue by ProCode-modder** [Oifox/Auto-Aim-Standoff-2#1]
not ur code this is procode's code dumb leech

**Comment by ImSevol** [comment]
standoff 2 aim source from old post on iosgods. Not your and not his

**Comment by ProCode-modder** [comment]
nope. i created the player.update without crashes using PlayerController and using aimbot without Update

**Comment by Oifox** [comment]
Its not ur code, i founded it on iosgods and recode it. I'm waiting for you to give me proofs that you did it

**Comment by ImSevol** [comment]
забавно. Тут все русские и все шото корчат из себя

**Comment by Oifox** [comment]
Why are u think that I russian?

**Comment by Oifox** [comment]
Стоп у меня же в профиле рашан девелопер

**Comment by ProCode-modder** [comment]
> Its not ur code, i founded it on iosgods and recode it. I'm waiting for you to give me proofs that you did it

Ебать ты меня рассмешил. Давай посмотрим)
Сразу же как сева написал что код есть на Iosgods, ты сразу же у нас подхватился)
![image](https://user-images.githubusercontent.com/81033830/125176560-02724f80-e1dd-11eb-8934-4c6c98638a95.png)
Это ты тоже сам накодил (чесно с моего сурса не спастил)
На iosgods нужен отдельный хук на rotationcontrol, чего нет у меня и в моем сурсе.
GetPlayerTeam, GetPlayerHealth, IsLocal - Тоже спизжено с сурса)
Пруфы приложил ниже
![image](https://user-images.githubusercontent.com/81033830/125176586-351c4800-e1dd-11eb-9569-6d56df957e00.png)
![image](https://user-images.githubusercontent.com/81033830/125176588-3baabf80-e1dd-11eb-908c-2ecf4cfd9e2c.png)
Забавно что указатели в таком же порядке как и в моем сурсе
![image](https://user-images.githubusercontent.com/81033830/125176596-541ada00-e1dd-11eb-95de-48a269d16624.png)
Тоже не спизжено))))
![image](https://user-images.githubusercontent.com/81033830/125176601-5aa95180-e1dd-11eb-955f-9a5fdebbcae2.png)
Все что на твоем гите - просто спизжено.
Ждем все пруфы что ты все сам сделал

**Comment by ProCode-modder** [comment]
Сева кста ты заебал ныть, в тг напиши, поговорим уже нормально

**Comment by ProCode-modder** [comment]
![image](https://user-images.githubusercontent.com/81033830/125176675-081c6500-e1de-11eb-835b-9081f0d2ab47.png)
ХАЗВХЫАХЫВАХЫХВАХВХЫАХХАХАХАХАХА

**Comment by ProCode-modder** [comment]
ебать слился

**Comment by ProCode-modder** [comment]
u have 24h to delete all pasted code from github or i will dmca u

**Comment by ProCode-modder** [comment]
ты же наш келпи, надо бы тебя задоксить

**Comment by Oifox** [comment]
Да я слился, а что?

**Comment by ProCode-modder** [comment]
удачи тебе дальше все пиздить школьник ебучий)
![image](https://user-images.githubusercontent.com/81033830/125187673-6c204700-e239-11eb-8c84-b2b857f06964.png)

**Comment by ProCode-modder** [comment]
ебать я проорал с тебя

**Comment by Oifox** [comment]
Я не говорил что это мой код, я просто делюсь с остальными кодом, который вряд-ли кто-то может найти

**Comment by ProCode-modder** [comment]
ебать угараю с ребенка
![image](https://user-images.githubusercontent.com/81033830/125189856-e86b5800-e242-11eb-8922-d856967cd76d.png)

**Comment by Oifox** [comment]
Зачем ты меня трогаешь? Я тебе ничего не делал, я просто помогаю beginer в разработке

**Comment by ProCode-modder** [comment]
ты мои коды воруешь и на гит заливаешь чтобы всякие скриптодети по типу тебя их пиздили

**Comment by h1ghc0d3** [comment]
лашки
